modules/measurement/budget-calculator/load.php

```diff
@@ -0,0 +1,165 @@
+<?php
+/**
+ * Module Name: Budget Calculator
+ * Description: Allows users to define their own metrics and generate a budget.json file from them.
+ * Experimental: No
+ *
+ * @package performance-lab
+ * @since 1.0.0
+ */
+
+/**
+ * Display the content of the budget calculator option page.
+ *
+ * @since 1.0.0
+ */
+function budget_calc_page_content() {
+	echo '<div class="wrap">
+	<h1>Budget Calculator</h1>
+	<form method="post" action="options.php">';
```

Comment:
A `nonce` here would be great.

Reply:
There is already one generated by calling `settings_field` below 🙂  
![Screenshot 2022-03-01 at 14 26 45](https://user-images.githubusercontent.com/2980676/156110429-113a1306-2450-4b3a-9f6f-0752805ccdd9.png)

It's also already verified on submission too (tried changing the value of the field and I get a "link expired" error from WP)